packages/foreman/rubygem-kafo/rubygem-kafo.spec

```
@@ -25,7 +25,7 @@ Requires: %{?scl_prefix}rubygem(logging) < 3.0.0
 Requires: %{?scl_prefix}rubygem(clamp) >= 0.6.2
 Requires: %{?scl_prefix}rubygem(clamp) < 1.3.1
 Requires: %{?scl_prefix}rubygem(highline) >= 1.6.21
-Requires: %{?scl_prefix}rubygem(highline) < 2.0
+Requires: %{?scl_prefix}rubygem(highline) < 2.1.0
```

Looks like the proper update procedure wasn't followed. Updating to 6.2.0 should have updated this
```suggestion
Requires: %{?scl_prefix}rubygem(highline) < 3.0.0
```

It has the correct markings so `bump_rpm.sh` should have caught this. If you used obal to bump and it doesn't do this, IMHO it means obal should not be used to bump gem RPMs until it's fixed to update dependencies.